packages/react-devtools-extensions/src/__tests__/__source__/ComponentUsingHooksIndirectly.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import React, {useEffect, useState} from 'react';
+
+export function Component() {
+  const countState = useState(0);
+  const count = countState[0];
+  const setCount = countState[1];
+
+  const darkMode = useIsDarkMode();
+  const [isDarkMode, setDarkMode] = darkMode;
+
+  useEffect(() => {
```

Comment:
What's the empty `useEffect` ?

Reply:
note that this file is just a fixture for a test. we use it in this fixture and assert that we don't generate a name for this hook, since naming hooks like useEffect doesn't make sense.

Reply:
Oh I get it that confirming shouldn't generate name, thanks!